strawberryfields/compilers/gaussian_unitary.py

```diff
@@ -18,18 +18,70 @@
 from strawberryfields import ops
 from strawberryfields.parameters import par_evaluate
 from thewalrus.symplectic import (
-    expand_vector,
     expand,
     rotation,
     squeezing,
     two_mode_squeezing,
     interferometer,
     beam_splitter,
 )
-
 from .compiler import Compiler
 
 
+def _apply_symp_one_mode_gate(S_G, S, r, i):
+    """In-place applies a one mode gate G to the symplectic operation, S, in mode i
+    Args:
+        S_G (array): 2x2 matrix for one mode symplectic operation
+        S (array): 2Mx2M Symplectic operation
+        r (array): M length means vector
+        i (int): index of one mode gate
+
+    Returns:
+        S (array): updated symplectic operation
+        r (array): updated means vector
```

Comment:
Python functions can only return a single object --- in this case, it is returning a tuple :)
```suggestion
    Returns:
        tuple[array, array]: returns a tuple containing the updated symplectic operation
        and the updated means vector
```

Reply:
I'm happy to make this change, but thought I'd just highlight that I've seen multiple object return docstrings elsewhere (e.g. https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.subplots.html), even though python does pack them into a tuple.

Reply:
Oh, that's good to know! The reason we use by convention one return in the docs here is that I remember there being a rendering issue if we tried to return more than one in the past.

This could be because we use Google-style docstrings, rather than NumPy-style? Or it could have been an issue in an old version of Sphinx which is now rectified.

Reply:
yeah, maybe good to stick with the local convention here.